Neos.ContentRepository.BehavioralTests/Classes/TestSuite/Behavior/GherkinPyStringNodeBasedNodeTypeManagerFactory.php

```diff
@@ -60,8 +57,18 @@ public function getLabel(Node $node): string
         );
     }
 
+    public static function registerNodeTypeConfigurationForContentRepository(ContentRepositoryId $contentRepositoryId, array $nodeTypeConfiguration): void
+    {
+        file_put_contents(self::cacheFileName($contentRepositoryId), json_encode($nodeTypeConfiguration, JSON_THROW_ON_ERROR));
+    }
+
     public static function reset(): void
     {
-        self::$nodeTypesToUse = null;
+
+    }
+
+    private static function cacheFileName(ContentRepositoryId $contentRepositoryId): string
+    {
+        return sys_get_temp_dir() . 'nodeTypesConfiguration_' . $contentRepositoryId->value . '.json';
```

Comment:
sys_get_temp_dir() is without trailing slash, there should be DIRECTORY_SEPARATOR in there afterwards.

Reply:
or we just use `FLOW_PATH_TEMPORARY` so its easier to debug it  when it fails :D ... i know we technically dont have a depenendcy but we do as we do use the content repository registry for tests. and that is fine. if someone wants to run the tests without flow he can improve that ;)